images/router/haproxy/Dockerfile.centos7

```diff
@@ -0,0 +1,32 @@
+#
+# This is the HAProxy router for OpenShift Origin.
+#
+# The standard name for this image is openshift/origin-haproxy-router
+#
+FROM openshift/origin
+
+#
+# Note: /var is changed to 777 to allow access when running this container as a non-root uid
+#       this is temporary and should be removed when the container is switch to an empty-dir
+#       with gid support.
+#
```

Comment:
```
-RUN yum -y install haproxy && \
+RUN INSTALL_PKGS="haproxy" && \
+    yum install -y $INSTALL_PKGS && \
+    rpm -V $INSTALL_PKGS && \

```
There is no rpm verification in the plain Dockerfile
I was planning on going back and fixing the plain Dockerfile, and then remove the Dockerfile.centos7, unless you'd rather I do them both in this one pull request.